src/xcode/ENA/ENA/Source/Scenes/SelectValueViewController/SelectableValue.swift

```diff
@@ -0,0 +1,27 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+struct SelectableValue: Comparable {
+	
+		// MARK: - Internal
+	let title: String
+	let subtitle: String?
+	let identifier: String?
+	let isEnabled: Bool
+	
+		// MARK: - Init
```

Comment:
The init section should be above the internal section, empty line after marks are missing and the marks are inset on tab to much